rocky/rocky/views/task_detail.py

```diff
@@ -1,32 +1,41 @@
 from django.contrib import messages
-from django.http import JsonResponse
+from django.http import FileResponse, HttpResponse, JsonResponse
+from django.shortcuts import redirect
 from django.urls import reverse
 from django.utils.translation import gettext_lazy as _
 from django.views.generic import TemplateView
-from katalogus.views.mixins import BoefjeMixin, NormalizerMixin
 
-from rocky.scheduler import TaskNotFoundError, client
-from rocky.views.mixins import OctopoesView
+from rocky.scheduler import SchedulerError
+from rocky.views.tasks import SchedulerView
 
 
-class TaskDetailView(OctopoesView, TemplateView):
-    def get_task(self, task_id):
-        try:
-            return client.get_task_details(self.organization.code, task_id)
-        except TaskNotFoundError as error:
-            messages.error(self.request, error.message)
+class DownloadTaskDetail(SchedulerView):
+    def get(self, request, *args, **kwargs):
+        task_id = kwargs["task_id"]
+        filename = "task_" + task_id + ".json"
+        task_details = self.get_task_details(task_id)
```

Comment:
No try/catch around this get_task_details for bytes errors?

Reply:
Yes will be handled by the SchedulerView at: https://github.com/minvws/nl-kat-coordination/blob/ca3bf8c05933b7c3ecd1639734c62325aa93bcad/rocky/rocky/views/scheduler.py#L62